src/main/java/io/github/thebusybiscuit/slimefun4/implementation/setup/SlimefunItemSetup.java

```diff
@@ -1524,6 +1524,10 @@ public static void setup(@Nonnull SlimefunPlugin plugin) {
         new ItemStack[] {SlimefunItems.CARBONADO, SlimefunItems.REDSTONE_ALLOY, SlimefunItems.CARBONADO, new ItemStack(Material.REDSTONE), SlimefunItems.LARGE_CAPACITOR, new ItemStack(Material.REDSTONE), SlimefunItems.CARBONADO, SlimefunItems.REDSTONE_ALLOY, SlimefunItems.CARBONADO})
         .register(plugin);
 
+        new Capacitor(categories.electricity, 524288, SlimefunItems.ENERGIZED_CAPACITOR, RecipeType.ENHANCED_CRAFTING_TABLE,
+                new ItemStack[] {new ItemStack(Material.NETHER_STAR), SlimefunItems.REDSTONE_ALLOY, new ItemStack(Material.NETHER_STAR), new ItemStack(Material.REDSTONE), SlimefunItems.CARBONADO_EDGED_CAPACITOR, new ItemStack(Material.REDSTONE), new ItemStack(Material.NETHER_STAR), SlimefunItems.REDSTONE_ALLOY, new ItemStack(Material.NETHER_STAR)})
```

Comment:
4 Nether stars is quite expensive for something that can be substituted with 8 carbonado capacitors.

Reply:
Well I think carbonado is more expensive and grindy. But since all the other recipes required 4 pf the previous tier. Maybe make the recipe 4 carb capacitor and perhaps 1 nether star.

Reply:
So carbonara instead of stars and stars instead of redstone?

Reply:
Check the new commit. Good enough?